src/dawn/engine_dawn.cc

```diff
@@ -488,6 +513,8 @@ Result EngineDawn::CreateFramebufferIfNeeded(
     RenderPipelineInfo* render_pipeline) {
   Result result;
 
+  // First make the Dawn color attachment textures that the render pipeline will
+  // write into.
```

Comment:
I'm a bit confused about Amber now.

The Amber pipeline object has both a GetFramebufferWidth and GetFramebufferHeight. 
But it *also* has color attachments that have a BufferInfo and hence Buffer object inside, and that Buffer has both GetWidth and GetHeight.

So... which is authoritative.

Where I started wondering about this is that the Amber pipeline also has GetDepthBuffer... should we be using its GetWidth and GetHeight?

I'm approving this use of width and height.  But I'll file a separate issue to @dj2 to explain what the correct usage should be.   In a world where we have multiple colour attachments, it seems weird to use one single pair of (width,height) for all those colour attachments.

Reply:
I filed https://github.com/google/amber/issues/468 to resolve this question.

Reply:
The GetFramebufferWidth and Height are authorative. The bufferinfo width/height are a communication channel back to the embedder if buffer extractions were requested.

The depth buffer, i'm assuming has to match the colour attachment sizes so should also use the GetFramebufferWidth/Height sizes.

As far as I understand, don't all colour attachments have to have the same size? (at least in vulkan?)

Reply:
@dneto0 Are all colour attachments in Vulkan the same size?